README.md

```diff
@@ -1,5 +1,6 @@
 # Examples
-Public facing examples for populating Saturn Cloud installations
+
+Public facing examples for populating Saturn Cloud installations.
 
 ## Structure
```

Comment:
I think we should just reference contributing from here instead of duplicating content

Reply:
ok can do. Let's wait until #6 is agreed and then I'll merge that into this